model/subnet_metadata.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package model
+
+// SubnetMetadata is the subnet metadata.
+type SubnetMetadata struct {
+	AccountID  string
+	VPCID      string
+	VPCPeering bool
+}
```

Comment:
Hey Szymon the plan here is to add some metadata to the VPC CIDR when its associated with an account so that we see some info. But maybe you are right, we need sth else if we want to filter by peering or other info. Do you have any suggestions on how to do that?

Reply:
Discussed in DMs